keyboards/sepsepkbd/yubino_hara/readme.md

```diff
@@ -0,0 +1,32 @@
+# yubino_hara
+
+![yubino_hara](https://raw.githubusercontent.com/nagmeal/sep-sep-kbd/main/yubino-hara/yubinohara.jpg)
```

Comment:
Please rehost on Imgur or similar image hosting site.

Reply:
Ok, I'll upload it to Imgur.